server/api/person/person.subscribe.js

```diff
@@ -28,10 +33,28 @@ module.exports = (server) => {
     PubSub.publish(TOPIC_PERSON__EMAIL_SENT, info)
   })
 
+  PubSub.subscribe(TOPIC_PERSON__CREATE, async (msg, person) => {
+    const coreOrganisation = await Organisation.findOne({ name: CORE_ORGANISATION_NAME }).lean().exec()
+
+    if (!coreOrganisation) {
+      return
+    }
+
+    await addMember({
+      person: person.id.toString(),
```

Comment:
I wonder whether this should be person._id?

person.id would likely return null and fail to save the member record.

Reply:
Have updated to `person._id.toSTring()`